jlibra-core/src/main/java/dev/jlibra/admissioncontrol/AdmissionControl.java

```diff
@@ -61,4 +61,9 @@ public UpdateToLatestLedgerResult updateToLatestLedger(Query query) {
 
         return GrpcMapper.updateToLatestLedgerResponseToResult(response);
     }
+
```

Comment:
Why is this toString method included in the pr? (mistake?)

Reply:
That would be great to have for AdmissionControl (that the client-readyness thing) - eg. I use it in unit tests in the spring-boot-starter and in the shell to print AdmissionControl - and toString is not bad to have.